dashboard/dashboard.py

```diff
@@ -338,9 +338,63 @@ def summarize_experiment(info, experiments_dir, exp_name):
     info.report_exp_status(exp_name, 'summary', status)
 
 
+def start_telemetry(script_dir, exp_name, output_dir, interval=15):
+    '''
+    Start recording the telemetry info.
+    :returns: subprocess instance of the telemetry recorder
+    '''
+    return subprocess.Popen(['python3', f'{script_dir}/run.py', 
+                            f'--exp_name={exp_name}',
+                            f'--interval={interval}',
+                            f'--output_dir={output_dir}'], stdout=subprocess.PIPE)
+
+def process_telemetry_statistics(info, exp_name, output_dir, time_str):
```

Comment:
It would be nice to factor out the different parts of this process as well (in a separate file) to make it easier to understand and possibly more configurable down the line